components/memix/memix.app.mjs

```diff
@@ -0,0 +1,30 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
+export default {
+  type: "app",
+  app: "memix",
+  propDefinitions: {},
+  methods: {
+    async getTemplates() {
+      return await axios(this, {
```

Comment:
here it would be better using the $ parameter, instead of `this`. because pipedream's axios can export more readable errors with it

Reply:
sgtm